ui-bookly/components/feed.tsx

```diff
@@ -1,18 +1,95 @@
 'use client';
-import { Grid, Card } from '@mui/material';
+import { Grid, Card, TextField, Container } from '@mui/material';
 import * as React from 'react';
+import { useEffect, useState } from 'react';
+import { OrganisationCard } from './OrganisationCard';
+import { gql, useQuery } from '@apollo/client';
+
+type organisation = { 
+    id: number,
+    name: string
+}
+
+type organisations = organisation[];
 
 const Feed: () => React.ReactElement = () => {
+
+    const OrganisationCardList = () => {
+        if (organisations.length == 1 && !organisations[0].id) {
+            return
+        } else {
+            return (
+                <>
+                    <Grid sx={{pt: 2}} container spacing={2}>
+                        {organisations.map((organisation) => {
+                            return (
+                                <Grid key={organisation.id} sx={{width: 1}} item justifyContent="center" alignItems="center">
+                                    <OrganisationCard organisation={organisation}/>
+                                </Grid>
+                            )
+                        })}    
+                    </Grid>
+                </>
+            )
+        }
+    };
+
+    const GET_ORGANISATIONS = gql`
+        query GetOrganisations {
+            organisations {
+                id
+                name
+            }
+        }
+    `;
+        
+    const { loading, error, data } = useQuery(GET_ORGANISATIONS)
```

Comment:
I could be wrong but you might need to run this in a useEffect to avoid a double render

Reply:
It seems to only render once

Reply:
Ah yeah, useQuery doesn't need to need a useEffect, I found that out on an unrelated project yesterday